mcs/build/config-default.make

```diff
@@ -18,7 +18,7 @@ TEST_RUNTIME_FLAGS =
 endif
 
 TEST_HARNESS = $(topdir)/class/lib/$(PROFILE_DIRECTORY)/$(PARENT_PROFILE)nunit-lite-console.exe
-PLATFORM_DEBUG_FLAGS = /debug:portable
+PLATFORM_DEBUG_FLAGS = /debug:embedded
```

Comment:
Be sure to test both/all, not delete one test to add another.

Reply:
Also, can these settings be combined -- portable and embedded?

Reply:
Ah, embedded implies portable.

Reply:
I removed this change, I will do this in another PR.